tools/gn/header_checker.cc

```diff
@@ -582,6 +582,7 @@ Err HeaderChecker::MakeUnreachableError(const InputFile& source_file,
 
   // Only display toolchains on labels if they don't all match.
   bool include_toolchain = !targets_with_other_toolchains.empty();
+  include_toolchain = true;
```

Comment:
Is this intended? If so, the initialization in line above is redundant.

Reply:
Nope. No idea how that joined the commit. It's from some debugging session long ago. I'll remove it.